javascript/javascript1/week2/homework-js1-w2.js

```diff
@@ -0,0 +1,168 @@
+// freeCodeCamp username: yuejiaoshi
+// freeCodeCamp profile url: https://www.freecodecamp.org/yuejiaoshi
+
+//*****************************************/
+// Flight booking fullname function
+//*****************************************/
+function getFullname(firstname, surname, useFormalName = false, gender) {
+  if (!useFormalName) {
+    return firstname + " " + surname;
+  } else {
+    if (gender === "man") {
+      return "Lord" + " " + firstname + " " + surname;
+    } else if (gender === "woman") {
+      return "Lady" + " " + firstname + " " + surname;
+    } else {
+      return (
+        "Please provide gender of " +
+        firstname +
+        " " +
+        surname +
+        " for a formal name," +
+        " 'man' or 'woman'?"
+      );
```

Comment:
You've used [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for the Student Manager task, but they could serve you very well here, too. 
```suggestion
      return `Please provide the gender of ${fullName} for a formal name man or woman?`;
```